### DISCUSSION TITLE
Unsplash images edit problem (pull #712 related)

### DISCUSSION THREAD

**Discussion by danielpetrica** [nuxt/image#719] [Q&A]
Hi everybody, please forgive me if I'm spamming but i recently started using this provider and I observed there was an edge case not handled with the Unsplash provider.
I opened a pull request https://github.com/nuxt/image/pull/712 which handles this case but there has been no action. 

Is there something I've done wrong or haven't done that blocks this pull request from been reviewed? 

The fact that images are not correctly formatted when passing an url instead of a path is a big problem as I'm using the url of the image returned by the Unsplash api is easier than obtaining the path, plus it keeps the tracking url parameters Unsplash uses for tracking which app produced views and downloads.

**Comment by DavidDeSloovere** [comment]
Your PR got merged, congrats.
Would you mind closing this question?

**Reply by danielpetrica** [reply]
Yes. I'm sorry for forgetting.
